docs/iot/TOC.md

```diff
@@ -2,10 +2,16 @@
 
 # Getting Started
 ## [Tizen IoT](/iot/getting-started/overview.md)
-## [Installing Tizen Studio](/iot/getting-started/tizen-studio-install.md)
-## [Flashing Tizen Images](/iot/getting-started/tizen-image-download-flash.md)
```

Comment:
Are you remove this /tizen-image-download-flash.md/?

Reply:
Yes. The contents of tizen-studio-install.md, tizen-image-download-flash.md and hardware-configuration.md move to many files, i.e. artik-sd-5.0.md, artik-no-sd-5.0.md, rpi3-5.0.md. and so on.

Reply:
Thank you for finding these broken links. 
I corrected the links and uploaded PR #448.
- tizen-studio-install.md --> #Installing-Tizen_studio
- tizen-image-download-flash.md --> setting-up-board.md or iot-preview-2.md
- hardware-configuration --> setting-up-board.md or iot-preview-2.md